client/src/pages/colors.js

```diff
@@ -0,0 +1,122 @@
+export const colors = [
```

Comment:
This file is completely unnecessary! I suppose this is for the colour manipulation for each cell. It can be achieved differently. HINT: use DOM manipulation with the specified library  @simonwep/selection-js library: https://simonwep.github.io/selection/. Study the library documentation.